WordPress/gutenberg
# Login/out: add spacing support

## What?

Adds margin and padding support to the login/out block.

## Why?

For consistency. Part of https://github.com/WordPress/gutenberg/pull/45147.

## How?

The PR adds spacing support to block.json.

## Testing Instructions

Make sure that the active theme supports spacing, for example, in theme.json.
Create a new post or page.
Add login/out block.
Confirm that there is a dimension panel, and that the margin and padding settings are available but not enabled by default.
Test the margin and padding settings in the editor and front.

-Go to the Site Editor, add a login/out block, and change the margin and padding via the block settings in the Styles sidebar.
-Adjust the spacing and margin via theme.json.

Example theme.json code: add this to styles > blocks:

```
"core/loginout": {
				"spacing": {
					"padding": {
						"top": "15px",
						"right": "15px",
						"bottom": "15px",
						"left": "15px"
					},
					"margin": {
						"top": "150px",
						"right": "15px",
						"bottom": "150px",
						"left": "15px"
					}
				}
			},
```

## Screenshots or screencast 

https://github.com/WordPress/gutenberg/assets/7422055/8b985ecc-87fe-44e4-8b74-5d34c1223007
